src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/search/parameter/SearchParameterKey.java

```diff
@@ -8,9 +8,10 @@ public enum SearchParameterKey {
     USER("user"),
     JURISDICTION("jurisdiction"),
     STATE("state"),
-    TASK_ID("taskId"),
-    TASK_TYPE("taskType"),
-    CASE_ID("caseId"),
+    TASK_TYPE_CAMEL_CASE("taskType"),
+    TASK_TYPE("task_type"),
+    CASE_ID_CAMEL_CASE("caseId"),
+    CASE_ID("case_id"),
     //R2 should be snake_case only,
```

Comment:
Maybe we should update the comment at the top of the file to say something like:
`If adding a new field please ensure they are added as snake_case only and not camelCase. The camelCase format is deprecated and will be removed.`

Reply:
will do